### DISCUSSION TITLE
TIMBC

### DISCUSSION THREAD

**Discussion by medhay36** [acroucher/PyTOUGH#49] [General]
Hello dear PYTOUGH  community, 

Does anyone have an idea how to generate a TIMBC block to the input file ?
I have an excel file composed of two columns (time and pressure) to be considered as time boundary conditions ? 

and thank you in advance

**Comment by acroucher** [comment]
hi, TIMBC was introduced in TOUGH3 and is not yet supported by PyTOUGH, unfortunately.

PyTOUGH supports all the input file blocks in TOUGH2 and AUTOUGH2 but there are a number of extensions introduced in TOUGH3 which haven't been implemented in PyTOUGH yet.
